### ISSUE TITLE
create prominent path to compliance dashboard from homepage 

### ISSUE THREAD

**Issue by gbinal** [GSA/datagov-wptheme#529]
Regardless of where the [project open data dashboard](https://labs.data.gov/dashboard) lives, I think it would make sense to be prominently accessible through the homepage.  

cc @rypan, @jlberryhill, @philipashlock @rebeccawilliams

**Comment by rebeccawilliams** [comment]
Three notes: 
1. Does this make sense as part of Federal Agency Participation?
2. We've been reviewing how to better organize the current header and footer menu items on the homepage in usability testing, so those should be changing in the near future to better reflect what users feel are important. 
3. Here are some other examples of dashboards combined with other information on open data sites for what they are worth: 
   - https://data.cityofnewyork.us/dashboard
   - https://cityofphiladelphia.github.io/slash-data/census/

**Comment by ErieMeyer** [comment]
What's the user need here?

**Comment by hkdctol** [comment]
@rebeccawilliams The discussion yesterday was leaning toward having it move to cio.gov since that's where Project Open Data is now.

**Comment by rebeccawilliams** [comment]
@hkdctol Right! I think @gbinal was flagging here that there should be a **link** to the dashboard URL that is navigable to from the Data.gov homepage.

Recent usability testing has provided feedback that (1) the current Footer menu is overwhelming in its number of options, and simultaneously often not noticed (see also #141) and (2) Open Data Compliance was ranked as an Important element for Data.gov (by 66% of participants), but not as high as Federal Agency Participation or Open Gov tools.  

Which is to say, I think linking to the dashboard somewhere totally makes sense, but where and how from the homepage (given the feedback above) should be a part of a larger navigation discussion in #141.

**Comment by rebeccawilliams** [comment]
This is currently linked to in the [FAQ](https://www.data.gov/metrics) and [Federal Agency Participation](https://www.data.gov/metrics) pages, but I agree could be more prominent. This should be considered in the Footer reorganization of #615, cc @hkdctol
